packages/sources/alphavantage-test/package.json

```diff
@@ -0,0 +1,42 @@
+{
+  "name": "@chainlink/alphavantage-test-adapter",
+  "version": "0.0.0",
+  "description": "Chainlink alphavantage-test adapter.",
+  "keywords": [
+    "Chainlink",
+    "LINK",
+    "blockchain",
+    "oracle",
+    "alphavantage-test"
+  ],
+  "main": "dist/index.js",
+  "types": "dist/index.d.ts",
+  "files": [
+    "dist"
+  ],
+  "repository": {
+    "url": "https://github.com/smartcontractkit/external-adapters-js",
+    "type": "git"
+  },
+  "license": "MIT",
+  "scripts": {
+    "clean": "rm -rf dist && rm -f tsconfig.tsbuildinfo",
+    "prepack": "yarn build",
+    "build": "tsc -b",
+    "server": "node -e 'require(\"./index.js\").server()'",
+    "server:dist": "node -e 'require(\"./dist/index.js\").server()'",
+    "start": "yarn server:dist"
+  },
+  "dependencies": {
+    "@chainlink/external-adapter-framework": "0.22.0",
```

Comment:
Just dropping a reminder here to bump this but we should maybe wait until Alejo's fw changes are in so we can go directly to that version.

Reply:
Version `0.25.0` is ready to be used

Reply:
Done